src/stream/src/executor/source.rs

```diff
@@ -276,7 +277,12 @@ impl<S: StateStore> SourceExecutor<S> {
                 .await
                 .map(SourceStreamReaderImpl::TableV2),
             SourceImpl::Connector(c) => c
-                .stream_reader(state, self.column_ids.clone())
+                .stream_reader(
+                    state,
+                    self.column_ids.clone(),
+                    self.metrics.registry.clone(),
```

Comment:
@wcy-fdu I had some trouble with building `SourceMetrics` by this registry. pls take a look